### PR TITLE
add checks for broken actions without a schedule

### LINKED ISSUE #8: per_page has no effect on number returned
Changing the per_page dropdown at the bottom of the table still returns the default 100 rows.

### PR DESCRIPTION
**Pull request by rrennick**
Closes #8 

This PR adds checks to handle trashed actions and actions which are assigned a NullSchedule in https://github.com/Prospress/action-scheduler/pull/292 .

### CODE REVIEW

**Comment by rrennick** [comment]
@psealock When you have a few minutes can you give this a test drive?

File: js/src/index.js [hunk 1]
```diff
@@ -24,3 +25,17 @@ addFilter( 'woocommerce_admin_reports_list', 'analytics/scheduled-actions', repo
 	  },
   ];
 } );
+
+/**
+ * Use the 'woocommerce_admin_time_excluded_screens' filter to remove date parameters from the query string.
+ */
+addFilter( 'woocommerce_admin_reports_list', 'analytics/scheduled-actions', pages => {
```
Comment (psealock) [inline js/src/index.js]: This is the same filter as the block above. Is this supposed to be the `woocommerce_admin_time_excluded_screens` filter?

If so, you'll just need to push `scheduled-actions` onto the array before returning it.

Comment (rrennick) [reply js/src/index.js]: This change should have been in #11. I have updated it here to use the correct filter and changed the unique filter names to better names.

**Comment by rrennick** [comment]
To create a broken schedule edit a schedule in the database to break the serialized value

before
```
+---------+---------+--------------------------+----------------------------------------------------------------------------------------------------------+
| meta_id | post_id | meta_key                 | meta_value                                                                                               |
+---------+---------+--------------------------+----------------------------------------------------------------------------------------------------------+
|   55676 |    5371 | _action_manager_schedule | O:30:"ActionScheduler_SimpleSchedule":1:{s:41:" ActionScheduler_SimpleSchedule timestamp";i:1560789687;} |
+---------+---------+--------------------------+----------------------------------------------------------------------------------------------------------+
```

Change the `s:41` to `s:40`

```
+---------+---------+--------------------------+----------------------------------------------------------------------------------------------------------+
| meta_id | post_id | meta_key                 | meta_value                                                                                               |
+---------+---------+--------------------------+----------------------------------------------------------------------------------------------------------+
|   55676 |    5371 | _action_manager_schedule | O:30:"ActionScheduler_SimpleSchedule":1:{s:40:" ActionScheduler_SimpleSchedule timestamp";i:1560789687;} |
+---------+---------+--------------------------+----------------------------------------------------------------------------------------------------------+
```

We see the broken schedules reported after some sites have been migrated between servers.

**Pull request merged**